mu-plugins/blocks/global-header-footer/postcss/header/search.pcss

```diff
@@ -25,11 +25,27 @@
 	}
 
 	& .wp-block-navigation-item__label {
-		display: none;
+
+		/* See @mixin hide-accessibly in wporg-news-2021 */
```

Comment:
It might be helpful to make it a mixin here too, so we can reuse it, and it's obvious which rules are part of it.

Not a big deal, though.

Reply:
Do we have a mixin plugin for postcss?

Reply:
I don't think so. There's https://github.com/postcss/postcss-mixins, but it's not a big deal if you don't think it's worth it

Reply:
Yeah, I don't think we need to add one. In most cases we're able to pass the core `screen-reader-text` as a className to the block, that just wouldn't work in this case.